app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -8,3 +8,30 @@ body {
   background-color: whitesmoke;
 }
 
+// TODO: Hadi- Move this to specific file to only be accessible where needed
+.rooms-card {
```

Comment:
This is duplicated in `rooms.scss`.

Reply:
https://github.com/hadicheaito1/greenlight/blob/deb2fccb6215f26f82fcd6012babd7123da92184/app/assets/stylesheets/rooms.scss
https://github.com/hadicheaito1/greenlight/blob/deb2fccb6215f26f82fcd6012babd7123da92184/app/assets/stylesheets/application.bootstrap.scss#L5

Reply:
You can change the className to something more generic `customer-info & rooms-card -> hoverable`